### DISCUSSION TITLE
Pressure vs hydrostatic pressure

### DISCUSSION THREAD

**Discussion by ShehabGamrah** [firemodels/fds#13929] [Q&A]
I have a question about air moving from high to low pressure. If I model an extract within a corridor and draw in air from a tall shaft, the pressure drop would cause air to move from the higher pressure at the top of the shaft to the lower pressure that occurs in reality. However, I don’t understand how hydrostatic pressure doesn’t affect the fluid flow, even though it’s significant. Considering hydrostatic pressure would counteract this pressure drop, but it doesn’t cause air to flow from the bottom to the top.

**Comment by drjfloyd** [comment]
You need a net force for motion. The force of gravity opposes the pressure
gradient

**Reply by ShehabGamrah** [reply]
So consider the top point of the natural shaft,
A, and the bottom point B, the damper at the corridor 20m below. Initially, point A is at 0Pa (atmospheric pressure),and point B at 240Pa due to the hydrostatic pressure (ρgh). With extraction happening at point B within the corridor, its pressure still does not lower than atmospheric pressure, meaning that point B would still have the higher pressure, so how does air move downwards? If air moves from higher to lower pressure, how can the dominant pressure force be due to extraction yet it is smaller than the hydrostatic? Does hydrostatic have no significance?

**Reply by drjfloyd** [reply]
You have an open top tank of water on the ground. The water at the bottom has hydrostatic pressure given by the height of the tank times the density of water times the acceleration due to gravity. If your tank were say the size of the Challenger Deep, the hydrostatic pressure at the bottom would be around 0.1 gigaPascals. Would you expect the fluid in this tank suddenly explode out of the top? Hopefully not. Why? Consider a parcle of fluid at the bottom. What is the net force on the parcel? It is zero. The pressure all around it is the same. There is nothing to cause it to move. 

Now put a hole in the bottom of the tank at that parcel of fluid. Now do you expect the parcel to move? Hopefully yes. Why? Well now the top of the parcel sees hydrostatic pressure but the bottom of the fluid sees the local atmospheric pressure which is 0.0001 gigaPascals at sea level.

**Comment by ShehabGamrah** [comment]
Thanks for this explanation. Let’s consider the CFD example. When we output pressure, it doesn’t account for the hydrostatic pressure through the natural shaft, which could be 240 pascals instead. As a result, we would see a 20-30 pascals output pressure. So, does hydrostatic pressure have no significance, even if there’s an opening at the bottom (similar to the opening in the water tank example, which is a corridor damper)?

**Reply by Er9y714** [reply]
To see the hydrostatic pressure, try monitoring `ABSOLUTE PRESSURE` quantity.
